packages/crypto/src/__tests__/MultisigSignature.spec.ts

```diff
@@ -0,0 +1,145 @@
+import { MultisigAddress } from '@helium/address'
+import { MultisigSignature } from '..'
+import { usersFixture } from '../../../../integration_tests/fixtures/users'
+
+describe('create', () => {
+    it('invalid if not enough signatures', async () => {
```

Comment:
Bunch of lint errors to fix here too.

Reply:
@tyler-whitman - What's the accepted way to run lint on this project? I run yarn lint but am getting way more errors than I would expect...

Reply:
Yeah there are a lot of older unresolved lint errors. Which IDE do you use? You should be able to integrate for specific files or you can run yarn lint on a specific file by changing the command in `./package.json` line 8. For example:

`"lint": "eslint --ext .ts ./packages/crypto/src/MultisigSignature.ts"`

Reply:
You can also auto fix a lot of them by adding `--fix ` to the command